bedrock/firefox/templates/firefox/new/trailhead/thanks-b.html

```diff
@@ -0,0 +1,107 @@
+{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
+{% endblock %}
+
+{% block site_header %}
+  {% with hide_nav_download_button=True %}
+    {% include 'includes/protocol/navigation/menu-firefox/index.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <div class="mzp-l-content">
+    <div class="mzp-c-notification-bar mzp-t-warning unsupported">
+      <p>
+        {% trans url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista' %}
+        You’re using an insecure, outdated operating system <a href="{{ url }}">no longer supported by Firefox</a>.
+        {% endtrans %}
+      </p>
+    </div>
+    <div class="mzp-c-notification-bar mzp-t-success auto-download">
+      {# fallback_url is replaced by the platform download link via JS, but if
+      something fails the user should still get a link to a working download path. #}
+      <p>
+        {% trans id='direct-download-link', fallback_url=url('firefox.all') %}
+        Your download should begin automatically. Didn’t work? <a id="{{ id }}" href="{{ fallback_url }}">Try
+          downloading again</a>.
+        {% endtrans %}
+      </p>
+    </div>
+  </div>
+
+  {% call call_out(
+  title=_('You’re almost done…'),
+  heading_level=1,
+  include_cta=True
+  ) %}
+  <div id="download-button-wrapper-desktop">
+    {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
+  </div>
+  <h2>{{ _('Install Firefox in 2 easy steps') }}</h2>
+  {% endcall %}
+
+  <section class="mzp-l-content">
+    <ol class="l-columns-two">
+      <li class="c-picto-block">
+        <h3 class="c-picto-block-title">{{ _('Click on the Firefox installer.') }}</h3>
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/firefox/new/trailhead/browser-status-bar.png') }}" alt="">
+        </div>
+        <div class="c-picto-block-body">
+          <p><strong>{{ _('Look down, look left.') }}</strong> {{ _('Click on the Firefox installer in the lower bottom
+            left-hand corner.') }}</p>
+        </div>
+      </li>
+
+      <li class="c-picto-block">
+        <h3 class="c-picto-block-title">{{ _('Select ‘Yes.’') }}</h3>
+        <div class="c-picto-block-image">
+          <img src="{{ static('img/firefox/new/trailhead/browser-modal.png') }}" alt="">
+        </div>
+        <div class="c-picto-block-body">
+          <p>
+            {% trans %}
+            Click YES for Firefox. <strong>Select YES</strong> on the pop-up window to finish the install.
+            {% endtrans %}
+          </p>
+        </div>
+      </li>
+    </ol>
+
+    <aside class="sumo-section">
+      <h3>{{ _('Complete your install and get the automatically private and wildly fast Firefox.') }}</h3>
+      <p>
+        {% trans sumo='https://support.mozilla.org/bm/kb/how-download-and-install-firefox-windows' %}
```

Comment:
Should this URL remove `/bm/`?
```suggestion
        {% trans sumo='https://support.mozilla.org/kb/how-download-and-install-firefox-windows' %}
```